### PR TITLE
added docs for jobber.py

### PR DESCRIPTION
**Pull request by bashirk**
## Description
This PR adds inline documentation to the jobber.py file.

## Relevant Issue
Issue #269 

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/NimbleEdge/EnvisEdge/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/NimbleEdge/EnvisEdge/blob/main/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [NimbleEdge Styleguide](https://numpydoc.readthedocs.io/en/latest/format.html)

### CODE REVIEW

File: fedrec/multiprocessing/jobber.py [hunk 1]
```diff
@@ -15,15 +15,55 @@
 
 class Jobber:
     """
-    Jobber class handles job requests based on job types
+    Jobber class executes the pipeline and
+    publishes the results. It handles job requests based on the job type
+    and can also be extended to handle other job types.
```
Comment (divyanshugit) [inline fedrec/multiprocessing/jobber.py:20]: Please align the text properly.

**Review by divyanshugit — CHANGES_REQUESTED** [review]

**Comment by divyanshugit** [comment]
Please update the file.

File: fedrec/multiprocessing/jobber.py [hunk 2]
```diff
@@ -15,15 +15,55 @@
...
 class Jobber:
     """
-    Jobber class handles job requests based on job types
+    The Jobber class executes the pipeline and publishes the results. It
+    handles job requests based on the job type and can also be extended
+    to handle other job types.
+    
+    The job type and the job arguments are defined in the job request.
+    The job request is a dictionary that contains the job type and the
+    job arguments.
```
Comment (divyanshugit) [inline fedrec/multiprocessing/jobber.py:24]: Can you please rephrase it?

**Review by divyanshugit — CHANGES_REQUESTED** [review]

File: fedrec/multiprocessing/jobber.py (hunk 2 above)
Comment (bashirk) [reply fedrec/multiprocessing/jobber.py]: Done.

File: fedrec/multiprocessing/jobber.py [hunk 3]
```diff
@@ -15,15 +15,55 @@
...
 class Jobber:
     """
-    Jobber class handles job requests based on job types
+    The Jobber class is a singleton class that handles the batch scheduling
+    of job requests to run in sequence, based on the job type. It is a utility
+    program that executes the pipeline for the job requests and publishes
+    the results in the order in which the job requests are received.
+    
+    A job request is a dictionary that contains the job type and the
+    job arguments.
```
Comment (divyanshugit) [inline fedrec/multiprocessing/jobber.py:24]:
```suggestion
    the results in the order in which the job requests are received. A job 
    request a dictionary that contains the job type and the
    job arguments.
```

**Review by divyanshugit — CHANGES_REQUESTED** [review]

**Comment by divyanshugit** [comment]
LGTM!! @ramesht007

**Review by ramesht007 — APPROVED** [review]

**Pull request merged**